modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/exportimport/data/handler/VulcanBatchEnginePortletDataHandler.java

```diff
@@ -90,6 +95,50 @@ public boolean isCompany() {
 		return true;
 	}
 
+	protected Map<String, Serializable> buildParameterMap(
+		PortletDataContext portletDataContext) {
+
+		DateFormat dateFormat = DateFormatFactoryUtil.getSimpleDateFormat(
+			"yyyy-MM-dd'T'HH:mm:ss'Z'");
+
+		return HashMapBuilder.<String, Serializable>put(
+			"batchNestedFields",
+			() -> {
+				if (MapUtil.getBoolean(
+						portletDataContext.getParameterMap(),
+						PortletDataHandlerKeys.PERMISSIONS)) {
+
+					return "permissions";
+				}
+
+				return null;
+			}
+		).put(
+			"filter",
+			() -> {
+				StringBundler sb = new StringBundler(5);
+
+				if (portletDataContext.getStartDate() != null) {
+					sb.append("dateModified ge ");
+					sb.append(
+						dateFormat.format(portletDataContext.getStartDate()));
+				}
+
+				if (portletDataContext.getEndDate() != null) {
+					if (sb.length() > 0) {
+						sb.append(" and ");
+					}
+
+					sb.append("dateModified le ");
+					sb.append(
+						dateFormat.format(portletDataContext.getEndDate()));
+				}
+
+				return sb.toString();
```

Comment:
IMHO, we should avoid sending an empty filter if none of the dates have been set. We could be increasing the number of code executed in the Object's layer